user_service/services/email_service.go

```diff
@@ -0,0 +1,51 @@
+package services
+
+import (
+	"fmt"
+	"user_service/config"
+
+	"gopkg.in/gomail.v2"
+)
+
+type IEmailService interface {
+	SendEmail(token string, receiverEmail string) error
+}
+
+type EmailService struct {
+}
+
+func NewEmailService() IEmailService {
+	return &EmailService{}
+}
+
+func (es *EmailService) SendEmail(token string, receiverEmail string) error {
```

Comment:
Email service has a method send email.
Imagine this method needs to be called by some other dev in some other logic. He would be confused why he needs token to send email. 
Rather a more generic and reusable method could be:

```
SendEmail(toEmail string, content string) error
```